doc/input-output-reference/src/overview/group-zone-controls-thermostats.tex

```diff
@@ -117,6 +117,15 @@ \subsubsection{Inputs}\label{inputs-054}
 	End If
 \end{lstlisting}
 
+Note that this option should be used in caution. Following recommendations may be used to input the value of Temperature Difference Between Cutout And Setpoint:
+
+\begin{lstlisting}
+1. The heating and cooling setpoints must be separated by at least 2 times the Temperature Difference Between Cutout And Setpoint or there will be a fatal error.
+2. The thermostat setpoint not met hours are incorrect with this option.
+3. This option will cause excess operation of heat pump system supplemental heating coils.
+
+\end{lstlisting}
```

Comment:
Next time instead of a code `lstlisting` environment, use an `enumerate` environment.